ButterLocalServer/main.py

```diff
@@ -1,26 +1,40 @@
 from fastapi import FastAPI
+from pydantic import BaseModel
 import uvicorn
 import os
-
+import uuid
+import time
 
 app = FastAPI()
 
 
-@app.get("/test/a")
-def get_test_data_for_endpoint_a():
-    return {"movie_list": ["fast and the furious", "2 fast 2 furious"]}
+# Movies
+@app.get("/movies")
+def get_celeb_movies(name: str):
+	if name == "vin disel":
+		return {"movie_list": ["fast and the furious", "2 fast 2 furious"]}
+	else:
+		return {"movie_list": []}
+
+
+# Bungheads
+class BungheadRequestBody(BaseModel):
+	first: str
+	last: str
+	occupation: str
 
 
-@app.get("/test/b")
-def get_test_data_for_endpoint_a():
-    return {"movie_list": ["fast and the furious: tokyo drift", "fast 4"]}
+@app.post("/bungheads/apply/")
+def get_test_data_for_endpoint_a(person: BungheadRequestBody):
+	return {"identifier": uuid.uuid4(), "created_at": time.time(), "person": person}
 
 
-@app.get("/test/c")
-def get_test_data_for_endpoint_a():
-    return {"movie_list": ["fast 5", "2 fast 2 furious"]}
+# Time
+@app.get("/time")
+def get_time():
+	return {"time": time.time()}
 
 
 if __name__ == "__main__":
-    port = int(os.getenv("PORT", 8070))
-    uvicorn.run(app, host='localhost', port=port)
+	port = int(os.getenv("PORT", 8070))
+	uvicorn.run(app, host='localhost', port=port)
```

Comment:
is indentation in python a codestyling thing or a language requirement?

Reply:
it's a language requirement, i think i opened this in xcode and it messed up this indentation... i will do a separate PR to address indentation across the project